libraries/stdlib/src/kotlin/collections/Collections.kt

```diff
@@ -235,6 +235,28 @@ public fun <T> List<T>.binarySearch(fromIndex: Int = 0, toIndex: Int = size, com
 }
 
 /**
+ * Returns the elements from the list from indexes specified in range.
+ */
+operator fun <T> List<T>.get(indexesRange: IntRange): List<T> =
+        if (indexesRange.first < 0)
+            throw IndexOutOfBoundsException("Range beginning (${indexesRange.first}) is less than zero.")
+        else if (indexesRange.last > lastIndex)
+            throw IndexOutOfBoundsException("Range end (${indexesRange.last}) is bigger then the end of table ($lastIndex)")
+        else if (indexesRange.first <= indexesRange.last)
+            subList(indexesRange.first, indexesRange.last + 1)
```

Comment:
It should be `toList()` after because we need a copy but `subList` just refers to the original list.